tests/monotouch-test/SystemConfiguration/CaptiveNetworkTest.cs

```diff
@@ -74,11 +74,6 @@ public void TryCopyCurrentNetworkInfo ()
 				return;
 
 			Assert.AreEqual (StatusCode.OK, status, "Status");
-			// To get a non-null dictionary back, we must (https://developer.apple.com/documentation/systemconfiguration/1614126-cncopycurrentnetworkinfo)
-			// * Use core location, and request (and get) authorization to use location information
-			// * Add the 'com.apple.developer.networking.wifi-info' entitlement
-			// I tried this, and still got null back, so just assert that we get null.
-			Assert.IsNull (dict, "Dictionary");
```

Comment:
Since we don't know why this started I'd rather we check both conditions
* dict is null
* dict contains `en0`

That way our (manual / not-generated code) is still being called (and tested).

An alternative would be to check for `en0` only - and see if/where this fails so we might understand what's going on.

Reply:
The documentation for `CaptiveNetwork.TryCopyCurrentNetworkInfo` here might give some clues: https://developer.apple.com/documentation/systemconfiguration/1614126-cncopycurrentnetworkinfo

There's a lot of requirements to get network info, and if we don't satisfy them, we get null back. This also varies by iOS version, which may be another source of confusion.